mapping-engine/mapping-converter-javascript-android/pom.xml

```diff
@@ -0,0 +1,76 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.eclipse.vorto</groupId>
+        <artifactId>mapping-engine</artifactId>
+        <version>0.10.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>mapping-converter-javascript-android</artifactId>
+    <packaging>aar</packaging>
+    <properties>
+        <!--suppress UnresolvedMavenProperty -->
+        <android.sdk.path>${ANDROID_HOME}</android.sdk.path>
+        <maven.compiler.source>1.6</maven.compiler.source>

```

Comment:
If you are using 1.6 here, wouldnt you need to convert the model and model-runtime, required by the engine, too ? 
I cannot see this in the PR.